docs/topics/shared-messaging-work-queue-mission-intro-paragraph.adoc

```diff
@@ -0,0 +1,11 @@
+Mission proficiency level:
+//special case since topic is used by front end.
+ifdef::docs-topic[xref:proficiency_foundational[*{proficiency-foundational}*].]
```

Comment:
I 100000% disagree this is _foundational_. IMHO, it's _advanced_.

Reply:
changed proficiency to _advanced_